apis/go.mod

```diff
@@ -1,15 +1,15 @@
 module github.com/metal3-io/baremetal-operator/apis
 
-go 1.17
+go 1.19
 
 require (
-	github.com/google/uuid v1.1.2
-	github.com/metal3-io/baremetal-operator/pkg/hardwareutils v0.1.0
+	github.com/google/uuid v1.3.0
+	github.com/metal3-io/baremetal-operator/pkg/hardwareutils v0.1.2
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.7.0
-	k8s.io/api v0.24.2
-	k8s.io/apimachinery v0.24.2
-	sigs.k8s.io/controller-runtime v0.12.3
+	k8s.io/api v0.25.0
```

Comment:
Ugh, this conflicts with the CVE fix :( Isn't it a bit too early to pull 0.25 bits?